demo/admin/src/products/future/ManufacturersGrid.cometGen.ts

```diff
@@ -10,8 +10,13 @@ export const ManufacturersGrid: GridConfig<GQLManufacturer> = {
         { type: "text", name: "name", headerName: "Name" },
         { type: "text", name: "address.street", headerName: "Street" },
         { type: "number", name: "address.streetNumber", headerName: "Street number" },
-        { type: "text", name: "address.alternativeAddress.street", headerName: "Alt-Street" },
-        { type: "number", name: "address.alternativeAddress.streetNumber", headerName: "Alt-Street number" },
+        { type: "text", name: "address.alternativeAddress.street", headerName: "Alt-Street", tooltipMessage: "Street of alternative address" },
+        {
+            type: "number",
+            name: "address.alternativeAddress.streetNumber",
+            headerName: "Alt-Street number",
+            tooltipMessage: "Street number of alternative address",
```

Comment:
naming should be improved:

- this is only shown in header, so "header" should be included
- it renders a Info-Icon plus the tooltip on the info icon, I would include "Info"
- "Message" could be omitted, gives no additional context

Maybe `headerInfoTooltip`?